### PR TITLE
Created ability spec file for event organizer role

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
Added tests for abilities of an event organizer, added a new user josh with the event organizer role in the users.yml and modified the other fixtures for this user

### CODE REVIEW

**Comment by nikhilgupta1211** [comment]
@ChrisBr @bgeuken I have fixed the tests just the rubocop offenses are present please have a look at this. Thank you :)

**Comment by bgeuken** [comment]
I would say you can skip the rubocop here. The method is already that complex. So it's not a regression.

But definitely something we should refactor eventually

**Comment by nikhilgupta1211** [comment]
ok, thanks :)

File: app/models/ability.rb [hunk 1]
```diff
@@ -271,6 +271,79 @@ def initialize(user)
       can :read, Shipment
     end
 
+    #
+    # Event Organizer permissions
+    # -------------------
+    #
+    if role == 'event_organizer'
+      # Events
+      can [:read, :create], Event
```
Comment (bgeuken) [inline app/models/ability.rb]: This [seems](https://github.com/openSUSE/travel-support-program/blob/master/app/models/ability.rb#L67) to be covered by the regular user permissions.

**Comment by bgeuken** [comment]
@nikhilgupta1211 Did you check if we can reuse the defined abilities of one of the existing roles, eg. tsp_members. There might be some overlap

File: app/models/ability.rb (hunk 1 above)
Comment (nikhilgupta1211) [inline app/models/ability.rb]: yes thanks for pointing it out :)

**Comment by nikhilgupta1211** [comment]
Yes fixed it :)

File: spec/models/ability_for_event_organizer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,258 @@
+require 'spec_helper'
+require 'cancan/matchers'
+
+describe 'Event_organizer' do
+  fixtures :all
+
+  subject { Ability.new(user) }
+  let(:user) { users(:josh) }
```
Comment (bgeuken) [inline spec/models/ability_for_event_organizer_spec.rb]: Minor thing... I would write this in one line. There is no other code here that would use the user variable.

**Comment by bgeuken** [comment]
Looks fine to me. @ChrisBr please check as well:-)

I have one open question for the requester of the feature (https://github.com/openSUSE/travel-support-program/issues/31#issuecomment-316376540). Let's see if they reply soonish. If not we can just merge and update the code later.

**Comment by bgeuken** [comment]
Merged per discussion

**Comment by nikhilgupta1211** [comment]
@ChrisBr @bgeuken we should unmerge this as we are now not creating a global event organizer role.

**Pull request merged**